.github/workflows/trailing_commas.py

```diff
@@ -0,0 +1,66 @@
+"""This is an pretiffy script."""
+import os
+import sys
+import argparse
+
+
+def arg_parser_resolver():
+    """Parse the files of the directory."""
+    parser = argparse.ArgumentParser(
+        description='for parsing across the directory')
+    parser.add_argument('--dir', type=str, required=False, default=os.getcwd(),
+                        help='directory-location where files are present')
+    return parser.parse_args()
+
+
+def syntax_matcher(root: str, files: list):
+    """Patterm matching and adding trailing commas at the required places."""
```

Comment:
1. Please add better descriptions of the role of arguments passed to the functions in the docstrings. 
2. Add a methodology section at the top of the script
3. Explain what the main function does in the docstring.
4. Use [countline.py](https://github.com/PalisadoesFoundation/talawa/blob/master/.github/workflows/countline.py) as a guide